ui/src/tooltip.tsx

```diff
@@ -29,36 +30,36 @@ const
       flexGrow: 1
     },
     icon: {
-      color: '#323130',
+      color: cssVar('$text'),
       fontWeight: 400,
       fontSize: '14px',
       userSelect: 'none',
       textAlign: 'left',
       marginLeft: '0.5em',
       cursor: 'pointer'
+    },
+    preventOverflow: {
+      minWidth: 0,
     }
   })
 
 export const
-  XToolTip = ({ children, content, showIcon, expand }: {
+  XToolTip = ({ children, content, showIcon = true, expand = true }: {
     children: React.ReactChild,
     content?: S,
     showIcon?: B,
     expand?: B
   }) => {
     if (!content) return <>{children}</>
 
-    const
-      tooltipProps: Fluent.ITooltipProps = {
-        onRenderContent: () => (<div><Markdown source={content} /></div>)
-      }
+    const tooltipProps: Fluent.ITooltipProps = { onRenderContent: () => <div><Markdown source={content} /></div> }
     return (
       <div className={css.container} data-test='tooltip'>
         {
-          showIcon === undefined || showIcon
+          showIcon
             ? (
               <>
-                <div className={expand === undefined || expand ? css.element : ''}>{children}</div>
+                <div className={clas(css.element, expand ? '' : css.preventOverflow )}>{children}</div>
```

Comment:
Shouldn't it be 
```suggestion
                <div className={expand ? css.element : css.preventOverflow}>{children}</div>
```
according to the previous logic?

Are all the other form components checked for possible regressions? Both with and without tooltip.

Reply:
> Shouldn't it be according to the previous logic?

Reverted.

> Are all the other form components checked for possible regressions? Both with and without tooltip.

Only the ones that use `expand=false`.